docs/02-app/01-building-your-application/07-deploying/index.mdx

```diff
@@ -136,12 +136,14 @@ Next.js will automatically load the latest version of your application in the ba
 
 Sometimes you might want to run some cleanup code on process signals like `SIGTERM` or `SIGINT`.
```

Comment:
When self-hosting, you might want to run code when the server shuts down on `SIGTERM` or `SIGINT` signals.